### ISSUE TITLE
Transfer ownership?

### ISSUE THREAD

**Issue by shadowhand** [brianhaveri/Underscore.php#40]
@brianhaveri since it seems you have abandoned this package, would you consider transferring the ownership to someone else to maintain? I think that @thephpleague  would consider it over.

**Comment by olso** [comment]
+1

**Comment by guroot** [comment]
+1

**Comment by ginsterbusch** [comment]
+2

**Comment by simonbaudry** [comment]
+1

**Comment by olso** [comment]
Guys there is better looking alternative https://github.com/lstrojny/functional-php

**Comment by shadowhand** [comment]
Actually I've been actively working on https://github.com/Im0rtality/Underscore which is becoming quite decent.

**Comment by Xartok** [comment]
Another solution may be that a new maintainer forks the project and maintains the fork

**Comment by iam-merlin** [comment]
+1

**Comment by bluesnowman** [comment]
Here is an experimental functional library that is aimed at having stronger type safety: https://github.com/bluesnowman/fphp-saber

**Comment by shadowhand** [comment]
@bluesnowman fphp-saber really isn't comparable to underscore at all.

**Comment by bluetechy** [comment]
@shadowhand Agreed, `fphp-saber` takes more of a Scala / F# like implementation to functional programming, but I thought you might be interested in it since you have been working on your own version of a Underscore.js like implementation and may be looking for ideas on how to improve upon it or approach it.  I am open for comments on `fphp-saber` and am happy to provide you any feedback on your project (but will do that hereon out via your project).

**Comment by tejastank** [comment]
-1000

**Comment by gtgt** [comment]
I actually like shadowhand's alternative...

**Comment by mpetrovich** [comment]
I'm late to the party, but we built our own Underscore-like functional programming library since none of the tools out there were maintained, easy to use, or well-documented: https://github.com/nextbigsoundinc/dash

Highlights:
- Many data types supported: arrays, objects, Traversable, DirectoryIterator, and more
- Chaining
- Currying
- Lazy evaluation
- Custom operations
- Well-tested: Comprehensive tests with nearly 3,000 test cases and 100% code coverage

I hope it can be useful to someone. Feedback welcome!
